### ISSUE TITLE
There may be a missing label in the favicon home (probably for cursor overing)

### ISSUE THREAD

**Issue by Melkiades** [insightsengineering/cards#462]
✖ Icon "fa-home" lacks an `aria-label`.
ℹ Specify `aria-label` to make the icon accessible to screen readers.
ℹ Learn more in `vignette(pkgdown::accessibility)`.
